django/core/cache/backends/filebased.py

```diff
@@ -164,7 +164,5 @@ def _list_cache_files(self):
         Get a list of paths to all the cache files. These are all the files
         in the root cache dir that end on the cache_suffix.
         """
-        return [
-            os.path.join(self._dir, fname)
-            for fname in glob.glob1(self._dir, "*%s" % self.cache_suffix)
-        ]
+        cache_suffix_paths = os.path.join(self._dir, f"*{self.cache_suffix}")
+        return glob.glob(cache_suffix_paths)
```

Comment:
Should not we use `root_dir` instead? :thinking: 
```suggestion
        return glob.glob(f"*{self.cache_suffix}", root_dir=self._dir)
```

Reply:
That sounds good. I hadn't thought of that!